### PR TITLE
add sorting logic for submission tests by istrial and number

### PR DESCRIPTION
**Pull request by msmner**

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-results/SubmissionResults.tsx [hunk 1]
```diff
@@ -13,7 +13,15 @@ const SubmissionResults = ({ testRuns } : ISubmissionResultsProps) => {
...
-    const compareByOrderByAsc = (tr1: ITestRunDetailsType, tr2: ITestRunDetailsType) => tr1.orderBy - tr2.orderBy;
+    const compareByOrderByAsc = (tr1: ITestRunDetailsType, tr2: ITestRunDetailsType) => {
+        if(tr1.isTrialTest === tr2.isTrialTest)
+        {return tr1.orderBy-tr2.orderBy;}
+
+        if(tr1.isTrialTest)
+        {return -1;}
+
+        return 1;
+    };
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-results/SubmissionResults.tsx:24]: Rewrite with ternary

**Review by Minkov — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-results/SubmissionResults.tsx [hunk 2]
```diff
@@ -13,7 +13,15 @@ const SubmissionResults = ({ testRuns } : ISubmissionResultsProps) => {
         <TestRunDetails testRun={run} />
     ), []);
 
-    const compareByOrderByAsc = (tr1: ITestRunDetailsType, tr2: ITestRunDetailsType) => tr1.orderBy - tr2.orderBy;
+    const compareByOrderByAsc = (tr1: ITestRunDetailsType, tr2: ITestRunDetailsType) => {
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-results/SubmissionResults.tsx:16]: Method name should be updated as it sorts by test type and order by.

**Comment by shsimeonova** [comment]
Is eslint enabled in your project configuration?

**Review by shsimeonova — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-results/SubmissionResults.tsx (hunk 1 above)
Comment (msmner) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-results/SubmissionResults.tsx]: Done.

**Comment by msmner** [comment]
> 

It is set to automatic ESlint configuration

**Review by mishogv — APPROVED** [review]

**Review by Minkov — APPROVED** [review]

**Review by shsimeonova — APPROVED** [review]

**Pull request merged**